server/test/handlers/inputSchedule.test.js

```diff
@@ -0,0 +1,72 @@
+const appointmentHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing invalid user schedule input', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+
+    test('rejection of null values for worker and school ID', async () => {
+        // Arrange
+        const testworkerId = null;
+        const testschoolId = null;
+
+        // Act
+        const user = await workerTimeslotHandler.addWorkerTimeslot(testworkerID, testschoolID);
+
+        // Assert
+        expect(user).toBe(null);
```

Comment:
The current method is returning true instead of null for any and all inputted values so the tests would be failing.

Reply:
@catai9 would you be able to clarify your comment here?

Reply:
You created a stub which currently returns true regardless of the parameters passed in. Thus, any automated tests would fail as you are asserting them to be equal to null but the method is in fact returning true. I would suggest commenting out these lines (so they can be kept and referred to later on) but writing a general test that just tests that your method currently does return the intended value of true. Refer to #17 for an example.

Another side note is that the addWorkerTimeslot method returns a boolean value rather than an object so in line 14 (and other associated lines), I would recommend naming the variable to be isSuccessfullyInserted or another similar adjective to prevent confusion.